packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx

```diff
@@ -14,37 +14,49 @@ import End from './components/End';
 import Calendar from './components/Calendar';
 
 export interface IDatepickerRangeProps {
+  /**
+   * Applies locale-based formatting.
+   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
+   */
   locale?: string;
   /**
-   * The start date to display
+   * Sets the start date
    */
   startValue?: Date;
   /**
-   * The end date to display
+   * Sets the end date
    */
   endValue?: Date;
   /**
-   * Minimum value. Users are still able to manually enter a date below this value.
+   * Disables dates before this value on the calendar element
    */
   minValue?: Date;
   /**
-   * Maximum value. Users are still able to manually enter a date above this value.
+   * Disables dates after this value on the calendar element
    */
   maxValue?: Date;
   /**
-   * Returns the parsed start and end dates
+   * Handles start and end date changes
+   *
+   * @param {Object} values The selected start and end dates
```

Comment:
I could go with this, for sure. But I wonder if we should use the more thorough syntax like:

```
@param {Object} values The selected dates
@param {Date} [values.startValue] Start date
@param {Date} [values.endValue] End date
```

Or maybe ~The start date~ -> "Optional start date"?

Reply:
Can do. "Optional end date" as well?

Reply:
Whatever reads best to you.